AdblockPlusSafari/FilterList.h

```diff
@@ -19,19 +19,21 @@
 
 @interface FilterList : NSObject
 
-@property (nonatomic, strong, nullable) NSString *filename;
+@property (nonatomic, strong, nonnull) NSString *url;
+@property (nonatomic, strong, nonnull) NSString *fileName;
 @property (nonatomic, strong, nullable) NSString *version;
 
 @property (nonatomic) BOOL userTriggered;
 @property (nonatomic) BOOL downloaded;
 @property (nonatomic) BOOL updating;
 @property (nonatomic) BOOL lastUpdateFailed;
 @property (nonatomic) NSUInteger taskIdentifier;
+@property (nonatomic) NSUInteger updatingGroupId;
```

Comment:
As a clueless novice to the ABP code, i would like to have one line explanation of what is this number for. Task identifier would be nice to have explained too.